src/libraries/System.Linq.Expressions/tests/IndexExpression/IndexExpressionTests.cs

```diff
@@ -464,15 +464,15 @@ public void IndexedPropertyVarArgs()
             AssertExtensions.Throws<ArgumentException>("propertyName", () => Expression.Property(instance, "Item", Expression.Constant(0), Expression.Constant(0), Expression.Constant(0)));
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotLinqExpressionsBuiltWithIsInterpretingOnly))]
+        [Fact]
```

Comment:
Don't see ref emitting here and it passes with IL expression compiler disabled, so leaving like this.